src/store/reducers.ts

```diff
@@ -0,0 +1,18 @@
+import { combineReducers } from 'redux';
+import { connectRouter } from 'connected-react-router';
```

Comment:
Do we need redux for routing? Can we do it in a different PR?

Reply:
We need it to access routing properties in the component. It is super light redux for the router. I think it is no problem to keep it.